src/Appium.Net/Appium/Windows/WindowsOptions.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using static System.String;
+
+namespace OpenQA.Selenium.Appium.Windows
+{
+    public class WindowsOptions : AppiumOptions
+    {
+        private new string VendorPrefix = "ms";
+        private const string PlatformNameOption = "platformName";
+        private new const string AutomationNameOption = "automationName";
+        private new const string AppOption = "app";
+        private const string AppArgumentsOption = "appArguments";
+        private const string AppTopLevelWindowOption = "appTopLevelWindow";
+        private const string AppWorkingDirOption = "appWorkingDir";
+        private const string CreateSessionTimeoutOption = "createSessionTimeout";
+        private const string WaitForAppLaunchOption = "ms:waitForAppLaunch";
+        private const string ExperimentalWebDriverOption = "ms:experimental-webdriver";
+        private const string SystemPortOption = "systemPort";
+        private const string PreRunOption = "prerun";
+        private const string PostRunOption = "postrun";
+
+        public WindowsOptions()
+        {
+            PlatformName = "windows";
+            AutomationName = "windows";
+            AddKnownCapabilityName(AutomationNameOption, "AutomationName property");
```

Comment:
would it be necessary to update this list every time we add a new cap to the driver or users could do that on their own as well?

Reply:
It would be necessary to update this list every time we add a new cap. The base.addAdditionalOption() is advised for not yet added type safe options

Reply:
That goes against the purpose of Options classes in Selenium .Net, type safe options.

Reply:
WindowsOption.AddAdditionalOption(string optionName, string optionValue) See [here](https://github.com/SeleniumHQ/selenium/blob/ee3ecbda341528d23152bcc1ca9e590c4e60c10c/dotnet/src/webdriver/DriverOptions.cs#L225)